packages/domains-table/src/domains-table/domains-table-expires-renews-cell.tsx

```diff
@@ -14,11 +14,21 @@ function getNotice( {
 	expiryDate,
 	isAutoRenewing,
 	isExpired,
+	isHundredYearDomain,
 }: {
 	isAutoRenewing: boolean;
 	isExpired: boolean;
 	expiryDate: string;
+	isHundredYearDomain: boolean;
 } ): string {
+	if ( isHundredYearDomain ) {
+		return sprintf(
+			/* translators: %s - The date on which a domain has expired */
```

Comment:
```suggestion
			/* translators: %s - The date until which a domain was paid for */
```
Maybe this comment is more accurate?

Reply:
Good catch! Thanks @leonardost